### DISCUSSION TITLE
23.01 - configuration page error

### DISCUSSION THREAD

**Discussion by Kyetech-IT-Manager** [opendcim/openDCIM#1380] [Q&A]
fresh install on opensuse 15.5 server
Web LAMP pattern installed
php8 installed & rewrite loaded
(followed your RHEL9 install - nearly perfect)

login into openDCIM
click on 'Edit Configuration'

Chrome browser displays
This page isn’t working172.16.104.207 is currently unable to handle this request.
HTTP ERROR 500

/var/log/apache2/error_log 
[Thu Jun 15 10:55:10.437894 2023] [php:error] [pid 18001] [client 172.16.150.0:52044] PHP Fatal error:  Uncaught Error: Call to undefined function mime_content_type() in /opt/openDCIM/openDCIM-23.01/configuration.php:152\nStack trace:\n#0 /opt/openDCIM/openDCIM-23.01/configuration.php(373): BuildFileList()\n#1 {main}\n  thrown in /opt/openDCIM/openDCIM-23.01/configuration.php on line 152, referer: http://172.16.104.207/

Am I missing something? Or is this an issue?

**Comment by wilpig** [comment]
You're missing the php module that covers mimetype.

https://www.php.net/manual/en/function.mime-content-type.php

In your translation from the RHEL -> SUSE i'm sure the package names don't line up so you'll have to figure out what is missing.

The RHEL instructions were done from a minimal install so they make no assumptions on what is or isn't installed.

**Reply by Kyetech-IT-Manager** [reply]
Thank you

the opensuse 15 did not install the fileinfo by defaul

https://github.com/Icinga/icingaweb2/issues/4503

zypper install php8-fileinfo

and now the 'Edit Configuration' page works

**Reply by wilpig** [reply]
https://github.com/opendcim/openDCIM/wiki/OpenSUSE
